IN-global_conf.json

```diff
@@ -4,75 +4,76 @@
 		"clksrc": 1,
 		"clksrc_desc": "radio_1 provides clock to concentrator for most devices except MultiTech. For MultiTech set to 0.",
 		"antenna_gain": 0,
```

Comment:
I think the description is actually incorrect, and that this should be the antenna gain in dBd. That makes sense in conjunction with the maximum power output specified in ERP by ETSI. As far as I understand this value will be used to decrease the PA output power so that the ERP remains below the limit when using a high gain antenna.

In this case default value of 0 makes sense as a normal gateway setup will use a dipole antenna. Perhaps the antenna provided with a Multitech has a 3.55dBi gain.